packages/components/src/VirtualizedList/RowSelection/RowSelection.component.js

```diff
@@ -7,18 +7,21 @@ import theme from './RowSelection.scss';
  * Higher order row-renderer that wrap the provided row renderer.
  * It manages row selection classname and inject it to the row renderer props.
  */
-function getRowSelectionRenderer(WrappedComponent, { isSelected, getRowData }) {
+function getRowSelectionRenderer(WrappedComponent,
+	{ isSelected, isActive, getRowData, onRowClick }) {
 	function RowSelection(props) {
 		const rowData = getRowData(props);
 
 		const enhancedClassNames = classNames(
 			props.className,
 			theme['row-selection'],
-			{ [theme.selected]: isSelected(rowData) },
-			{ selected: isSelected(rowData) },
+			{ [theme.active]: isActive && isActive(rowData) },
+			{ active: isActive && isActive(rowData) },
+			{ [theme.selected]: isSelected && isSelected(rowData) },
+			{ selected: isSelected && isSelected(rowData) },
```

Comment:
Please extract the `isActive && isActive(rowData)` and `isSelected && isSelected(rowData)` tests in const and use them in `classNames()` to avoid testing multiple times